internal/model/model.go

```diff
@@ -1653,6 +1653,14 @@ func (m *Model) BringToFront(folder, file string) {
 // CheckFolderHealth checks the folder for common errors and returns the
 // current folder error, or nil if the folder is healthy.
 func (m *Model) CheckFolderHealth(id string) error {
+	// This could be expensive wrt sleeping disks... if random(1) < 0.2 ?
+	usage, syserr := osutil.DiskUsedPercentage(m.cfg.ConfigPath())
+	if syserr == nil && usage > 99 {
```

Comment:
syserr is an unusual variable name, would `if usage, err := osutil.DiskUsedPercentage(...); err == nil && usage > theConfigValue {` suffice here?